addons/cargo/stringtable.xml

```diff
@@ -314,12 +314,14 @@
         </Key>
         <Key ID="STR_ACE_Cargo_loadTimeCoefficient">
             <English>Load Time Coefficient</English>
+            <German>Ladezeitmultiplikator</German>
             <Japanese>積載時間の係数</Japanese>
             <Polish>Współczynnik czasu załadowania</Polish>
             <Italian>Coefficente Tempo Caricamento</Italian>
         </Key>
         <Key ID="STR_ACE_Cargo_loadTimeCoefficient_description">
             <English>Modifies how long it takes to load/unload items.\nTime, in seconds, is the size of the item multiplied by this value.</English>
+            <German>Gibt an, wie lange das (ent)laden von Gegenständen dauern soll.\nZeit in Sekunden, mit der die Größe des Items multipliziert wird.</German>
```

Comment:
I am not sure if "Gewicht" is better then "Größe" in this case. Opinions ?

Reply:
Das Laden / Entladen
Item sollte auch übersetzt werden

Reply:
mit der die Größe
->
die mit der Größe

That way the conjunction is placed right after the comma and the article is placed before the noun in genitive.

Reply:
better`?